### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wrathwolf**
Cleaned up README.md to make it more presentable for the "GitHub Include" to have clean information on the wiki.

I also updated the link to the meeting to migrate the BPA community meeting to the LF Zoom.

### CODE REVIEW

**Review by MarkusKuemmerle — CHANGES_REQUESTED** [review]
I would keep the content of the previous "Contributorship and mailing list" section. It explains how to subscribe and unsubscribe to the mailing list, and how to send emails to the group.

Please make sure that as soon as this PR is merged also the content of the website subpage is updated

**Review by hdamker — CHANGES_REQUESTED** [review]
Please comment in https://github.com/camaraproject/Template_Lead_Repository/pull/3 regarding the upcoming structure for the README.md. That should be agreed before we touch/change all repositories.

**Comment by PedroDiez** [comment]
Please, @wrathwolf could you address the comments?

File: README.md [hunk 1]
```diff
@@ -13,22 +14,21 @@ Repository to describe, develop, document and test the Blockchain Public Address
...
-    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml&nocors)
-    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
-
-## Contributorship and mailing list
-* To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit <https://lists.camaraproject.org/g/sp-bpa>.
-* A message to all Contributors of this Sub Project can be sent using <sp-bpa@lists.camaraproject.org>.
+## Contributing
+* Meetings
+  * Thursday, 13:00-14:00 CET/CEST (12:00 UTC, 11:00 UTC during European DST)
+  * [Registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/99588511751?password=c8764d4d-32b1-4aa6-91a5-a9cf74ee1aee)
```
Comment (PedroDiez) [inline README.md:21]: This link seems obsolete

File: README.md [hunk 2]
```diff
@@ -32,3 +29,12 @@ Repository to describe, develop, document and test the Blockchain Public Address
       - OpenAPI [YAML spec file](https://github.com/camaraproject/BlockchainPublicAddress/blob/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
       - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml&nocors)
       - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
+
+## Contributing
+* Meetings
+    * [Registration/Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/96846363178?password=9ebebe6c-c59c-42be-a709-7d5fd42fb626)
+    * Schedule: Bi-weekly on Thursdays, 13:00-14:00 CET/CEST (12:00 UTC, 11:00 UTC during European DST)
+    * Minutes: Access [meeting minutes](https://wiki.camaraproject.org/display/CAM/BlockchainPublicAddress).
```
Comment (hdamker) [inline README.md:37]: Suggest to point directly to meeting minutes and use a "share" link:

```suggestion
    * Minutes: Access [meeting minutes](https://wiki.camaraproject.org/x/iIAuAQ).
```

File: README.md [hunk 3]
```diff
@@ -13,22 +14,27 @@ Repository to describe, develop, document and test the Blockchain Public Address
...
+    * **The first available release and version of Blockchain Public Address API is 0.1.0.** There are expected changes in upcoming releases. It is addressed for early implementations, but not a stable version yet.
+    * Release 0.1.0 of the API is available within the [release-0.1.0 branch](https://github.com/camaraproject/BlockchainPublicAddress/tree/release-0.1.0):
+      - API definition **with inline documentation**:
+      - OpenAPI [YAML spec file](https://github.com/camaraproject/BlockchainPublicAddress/blob/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
+      - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml&nocors)
+      - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
```
Comment (hdamker) [inline README.md:31]: Proposal to align the Release Information with other APIs (I hope that the indentation will work as expected):

```suggestion
* Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.
* **The first initial release and version of Blockchain Public Address API is 0.1.0 and is available within the [release-0.1.0 branch](https://github.com/camaraproject/BlockchainPublicAddress/tree/release-0.1.0) ** There are expected changes in upcoming releases. It is addressed for early implementations, but not a stable version yet.
* API definition of Blockchain Public Address API is 0.1.0 (with inline documentation):
  - OpenAPI [YAML spec file](https://github.com/camaraproject/BlockchainPublicAddress/blob/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml&nocors)
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
* All releases and pre-releases are available here https://github.com/camaraproject/BlockchainPublicAddress/releases
* For changes see [CHANGELOG.md](https://github.com/camaraproject/BlockchainPublicAddress/blob/main/CHANGELOG.md)
```

File: README.md [hunk 4]
```diff
@@ -13,22 +14,22 @@ Repository to describe, develop, document and test the Blockchain Public Address
...
-* Release 0.1.0 of the API is available within the [release-0.1.0 branch](https://github.com/camaraproject/BlockchainPublicAddress/tree/release-0.1.0):
-  - API definition **with inline documentation**:
-    - OpenAPI [YAML spec file](https://github.com/camaraproject/BlockchainPublicAddress/blob/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
-    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml&nocors)
-    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
+* **The first initial release and version of Blockchain Public Address API is 0.1.0 and is available within the [release-0.1.0 branch](https://github.com/camaraproject/BlockchainPublicAddress/tree/release-0.1.0) ** There are expected changes in upcoming releases. It is addressed for early implementations, but not a stable version yet.
```
Comment (hdamker) [inline README.md:20]:
```suggestion
* **The first initial release and version of Blockchain Public Address API is 0.1.0 and is available within the [release-0.1.0 branch](https://github.com/camaraproject/BlockchainPublicAddress/tree/release-0.1.0)** There are expected changes in upcoming releases. It is addressed for early implementations, but not a stable version yet.
```

**Review by hdamker — APPROVED** [review]

**Comment by hdamker** [comment]
@wrathwolf 
I've finally and as a last step aligned the Release Information and dismissed the old change requests from @MarkusKuemmerle.
Up to @camaraproject/blockchain-public-address_codeowners to review, approve and merge

**Review by grgpapadopoulos — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -13,22 +14,22 @@ Repository to describe, develop, document and test the Blockchain Public Address
...
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/BlockchainPublicAddress/release-0.1.0/code/API_definitions/blockchain_public_address.yaml)
+* All releases and pre-releases are available here https://github.com/camaraproject/BlockchainPublicAddress/releases
+* For changes see [CHANGELOG.md](https://github.com/camaraproject/BlockchainPublicAddress/blob/main/CHANGELOG.md)
 
-## Contributorship and mailing list
-* To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit <https://lists.camaraproject.org/g/sp-bpa>.
-* A message to all Contributors of this Sub Project can be sent using <sp-bpa@lists.camaraproject.org>.
+## Contributing
+* Meetings
+    * [Registration/Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/96846363178?password=9ebebe6c-c59c-42be-a709-7d5fd42fb626)
```
Comment (PedroDiez) [inline README.md:30]:
```suggestion
    * [Registration/Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/99023344259?password=cb995600-56ee-48a2-ade1-607bb10487df)
```

Comment (PedroDiez) [reply README.md:30]: Think @wrathwolf latest link for Blockchain is this one

**Pull request merged**